src/langchain_google_alloydb_pg/engine.py

```diff
@@ -346,11 +368,16 @@ async def ainit_document_table(
     ) -> None:
         """
         Create a table for saving of langchain documents.
+        If table already exists, a TABLE_ALREADY_EXISTS error is thrown.
```

Comment:
Add a "Raises:" field after Args. Is the error formatted like "TABLE_ALREADY_EXIST" or is it an OperationalError with message the table already exists?

Reply:
i will rename this to `DuplicateTableError`, becuase the error is actually like below:
```
sqlalchemy.exc.ProgrammingError: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.DuplicateTableError'>: relation <table_name> already exists
[SQL: CREATE TABLE <full command>;]
```

```suggestion
        If table already exists, a DuplicateTableError error is thrown.
```

Reply:
Added raises as well.